flintrock/services.py

```diff
@@ -129,14 +130,14 @@ def install(
             command="""
                 set -e
 
-                python /tmp/download-hadoop.py "{version}"
+                python /tmp/download-hadoop.py "{version}" "{download_source}"
 
                 mkdir "hadoop"
                 mkdir "hadoop/conf"
 
                 tar xzf "hadoop-{version}.tar.gz" -C "hadoop" --strip-components=1
                 rm "hadoop-{version}.tar.gz"
```

Comment:
I'm a bit concerned about this since the file name could be not `hadoop-{version}.tar.gz` or, alternatively, we could impose the file name to be `hadoop-{version}.tar.gz` when `urlretrieve`ing, what do you think?